src/components/about/about.module.css

```diff
@@ -0,0 +1,41 @@
+.container {
+  display: flex;
+  flex-direction: column;
+  margin: 0 auto;
+  padding: 50px;
+}
+
+.img {
+  width: 330px;
+  height: 292px;
+  border-radius: 30px;
+  margin-bottom: 20px;
+}
+
+.title {
+  font-size: 2em;
+  margin-bottom: 10px;
+}
+
+.description {
+  line-height: 1.5;
+  width: 330px;
+}
+
+@media only screen and (min-width: 1280px) {
```

Comment:
Try to fix responsive
![Screenshot from 2024-04-18 16-59-16](https://github.com/legogoUA/fe-react-2024/assets/37895784/d7da6e59-b22a-44c7-8e41-3f40f8ae85a0)